### PR TITLE
many: move with some simplifications test snap building to snap/snaptest

### PR DESCRIPTION
**Pull request by pedronis**

### CODE REVIEW

File: snap/info.go [hunk 1]
```diff
@@ -245,8 +245,6 @@ func (app *AppInfo) ServiceSocketFile() string {
 
 // ReadInfo reads the snap information for the installed snap with the given name and given side-info.
 func ReadInfo(name string, si *SideInfo) (*Info, error) {
-	// XXX: test directly when we don't have to invent the nth way
```
Comment (zyga) [inline snap/info.go]: \o/

**Comment by zyga** [comment]
LGTM, thank you for improving MockSnap.

**Comment by pedronis** [comment]
retest this please

**Comment by pedronis** [comment]
retest this please

**Comment by mvo5** [comment]
Looks fine, thank you!

**Pull request merged**